libdevcore/LoggingProgramOptions.cpp

```diff
@@ -41,4 +41,27 @@ po::options_description createLoggingProgramOptions(unsigned _lineLength, Loggin
     return optionsDescr;
 }
 
+po::options_description createLoggingProgramOptions(unsigned _lineLength, LoggingOptions& _options, const std::string listLogChannels)
```

Comment:
You don't need to duplicate the whole function, you need just this one version and make the last parameter optional.
```suggestion
po::options_description createLoggingProgramOptions(unsigned _lineLength, LoggingOptions& _options, std::string const& logChannels = {})
```